tests/workbenches/utils.py

```diff
@@ -0,0 +1,221 @@
+#
+# Copyright Skodjob authors.
+# License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+#
+from __future__ import annotations
+
+import logging
+import time
+import traceback
+from contextlib import contextmanager
+from typing import Any
+
+from typing import Callable, Generator
+
+import kubernetes.dynamic
+from kubernetes.dynamic import DynamicClient, ResourceField
+
+import ocp_resources.pod
+import ocp_resources.resource
+
+
+class TestFrameConstants:
+    GLOBAL_POLL_INTERVAL_MEDIUM = 10
+
+
+class OcpResourceManager:
```

Comment:
What is the specific goal you have with this class? if you use resources as classes from ocp_resources you will have their lifecycle handled by the wrapper library directly

Reply:
removed OcpResourceManager